bears/image/ImageCompressionBear.py

```diff
@@ -0,0 +1,67 @@
+import os
+import optimage
+
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Result import Result
+from dependency_management.requirements.PipRequirement import PipRequirement
+from dependency_management.requirements.DistributionRequirement import (
+    DistributionRequirement)
+from coala_utils.FileUtils import create_tempfile
+
+
+class ImageCompressionBear(LocalBear):
```

Comment:
uhm... you should add, a short description of this bear, and a link to where more information can be fetch